src/client/style.scss

```diff
@@ -113,6 +114,7 @@ input:placeholder-shown + button[type="submit"], button.disabled {
 canvas {
   height: 100%;
   width: 100%;
+  object-fit: contain;
```

Comment:
Unfortunately Firefox has a bug with `object-fit: contain;` on webgl canvas, see: https://bugzilla.mozilla.org/show_bug.cgi?id=1686356

I'm not too sure how to address the issue, obviously `object-fit: contain;` is the desired behaviour, but I'm not sure I want to reimplement it in user-code just yet, would be best if we could find a workaround to the issue so that we can still use object-fit contain. If there is none, we'll probably have to deal with that manually in a `resize` handler and on startup.

If you have any idea of how to tackle this let me know. If not you can drop this commit for now and I'll fix it in a way that doesn't break on firefox at some later point :+1:

Reply:
I'm on Windows with AMD GPU - everything is ok in Firefox. Interesting, how fast they fix bugs..
But our page doesn't have bright background, it's always black, so flickering might be not visible. Can't test that, so I dropped commit.

Reply:
As you might imagine, I didn't go around digging for this bug because I was bored, I just experienced it while testing your branch. The bug report says it's a regression and it's only a month old, so maybe your version of Firefox is "too old" for this bug to show up. That would be funny :sweat_smile: